python/hail/methods/impex.py

```diff
@@ -272,10 +272,12 @@ def export_plink(dataset, output, call=None, fam_id=None, ind_id=None, pat_id=No
     l = dataset.locus
     a = dataset.alleles
 
+    cm_position0 = hl.float64(0.0) if cm_position is None else expr_or_else(cm_position, 0.0)
+
     bim_exprs = {'locus': l,
                  'alleles': a,
                  'varid': expr_or_else(varid, hl.delimit([l.contig, hl.str(l.position), a[0], a[1]], ':')),
-                 'pos_morgan': expr_or_else(position_morgan, 0)}
+                 'cm_position': cm_position0}
```

Comment:
Why can't you use `expr_or_else(cm_position, 0.0)`? That's what I wrote the function to do.

Reply:
I find this clearer because  if `cm_position=None` then there will be no missingness, so why are we checking? It's also more performant.

Reply:
it doesn't -- see the `expr_or_else` implementation

Reply:
ah! i conflated with `or_else`, good catch